docs/home/howto/MONITORING.md

```diff
@@ -231,50 +231,50 @@ When dashboard creation or import succeeds you will see it on your dashboard lis
 
 *Note: For some dashboards, there is no data to visualize until there is traffic activity for the monitored component.*
 
-# Kibana
+# OpenSearch Dashboards
 
-Kibana is an free and open frontend application that sits on top of the Elastic Stack, providing search and data visualization capabilities for data indexed in Elasticsearch. For more informations about Kibana please refer to [the official website](https://www.elastic.co/what-is/kibana).
+OpenSearch Dashboards ( a Kibana successor ) is an open source search and analytics visualization layer. It also serves as a user interface for many OpenSearch project plugins. For more information please refer to [the official website](https://opensearch.org/docs/latest/dashboards/index/).
```

Comment:
Is it really a Kibana successor? Please remove irrelevant opinions and write something like Kibana equivalent from ELK stack.